src/mono/sample/wasm/browser-bench/Console/Wasm.Console.Bench.Sample.csproj

```diff
@@ -5,7 +5,6 @@
     <WasmMainJSPath>$(BrowserProjectRoot)\test-main.js</WasmMainJSPath>
     <WasmGenerateRunV8Script>true</WasmGenerateRunV8Script>
     <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
-    <WasmNativeStrip>false</WasmNativeStrip>
```

Comment:
Will this produce some noise in graphs?

cc @radekdoulik

Reply:
I think we are not collecting graphs for V8 console, right ? Anyway, it only could become smaller and faster by this change.